repo/packages/A/artifactory-primary/1/config.json

```diff
@@ -0,0 +1,95 @@
+        },
+        "cpus": {
+          "description": "CPU shares to allocate to each Artifactory master.",
+          "type": "number",
+          "default": 2.0,
+          "minimum": 1.0
+        },
+        "mem": {
+          "description": "Memory to allocate to each Artifactory master.",
+          "type": "number",
+          "default": 2048.0,
+          "minimum": 1024.0
+        },
+        "high-availability": {
+          "description": "High availability mode properties.",
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "description": "Run Artifactory in high availability mode.",
+              "type": "boolean",
+              "default": false
+            },
+            "nodes": {
+              "description": "The number of nodes to run in high availability mode.",
+              "type": "number",
+              "default": 1
+            },
+            "unique-nodes": {
+              "description": "Run nodes on unique hosts.",
+              "type": "boolean",
+              "default": true
+            },
+            "cluster-home": {
+              "description": "The location of a volume on the host to be used for persisting cluster home directory used when running Artifactory in HA mode. The final location will be derived from this value plus the name set in `framework-name` (e.g. `/mnt/host_volume/artifactory`). Note that this path must be the same on all DCOS agents.",
+              "type": "string"
+            },
+            "database": {
+              "description": "High availability database properties.",
+              "type": "object",
+              "properties": {
+                "connection-string": {
+                  "description": "Database connection string.",
+                  "type": "string"
+                },
+                "user": {
+                  "description": "Database username.",
+                  "type": "string"
+                },
+                "password": {
+                  "description": "Database password.",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "artifactory-home": {
+          "description": "The location of a volume on the host to be used for persisting the Artifactory home directory. The final location will be derived from this value plus the name set in `framework-name` (e.g. `/mnt/host_volume/artifactory`). If high-availability mode is on, this location will also include the Mesos task ID to avoid overlapping. Note that this path must be the same on all DCOS agents.",
+          "type": "string",
+          "default": "/tmp"
+        },
+        "artifactory-primary-license": {
+          "description": "License to for artifactory primary",
+          "type": "string",
+          "default" : "NONE"
+        },
+        "artifactory-licenses": {
+          "description": "Licenses to popuplate artifactory nodes",
```

Comment:
`Licenses to popuplate artifactory nodes` -> `Licenses for Artifactory nodes`. How is this different to the primary license?